pages/login/index.js

```diff
@@ -42,7 +42,11 @@ export default function Login() {
 
   function authenticate(e) {
     e.preventDefault();
-    console.log(users.find)
+    users.forEach((user) => {
+      if(login.email == user.email) {
+        console.log(true) 
+      }
```

Comment:
**nitpick:** Revert this if statement to look something like this, it could reduce lines and remove wrapping.

```suggestion
      if (login.email != user.email) return
      console.log(true) 
```